### ISSUE TITLE
Android app

### ISSUE THREAD

**Issue by kornava** [farshid616/Android-TV-Remote-Controller-Qt#1]
Can this be used as ana Android App? any ideas how to do with opc on Android?

**Comment by farshid616** [comment]
No, unfortunately this app is not compatible to build for android devices. There are some dependencies like Qt opcua that is not available in Qt Android.
